nbconvert/exporters/slides.py

```diff
@@ -9,82 +9,101 @@
 from traitlets import Bool, Unicode, default
 
 from .html import HTMLExporter
+from ..preprocessors.base import Preprocessor
+
+
+class _RevealMetadataPreprocessor(Preprocessor):
```

Comment:
How does this now get associated to run for the slides exporter? I didn't see a change there to run this preprocessor by default unless I missed something?

Reply:
Also, should this be in preprocessors even though it's a private class? Maybe instead of being private we could put it inside a subdirectory `internal` under preprocessors so it's discoverable but indicates it's not nbconvert specific functionality? This would make any config assignments read a little nicer imo and fit the organization scheme a little more. Not fixed on the idea but thought I'd bring it up

Reply:
It gets associated in the constructor of the exporter.

Reply:
What I would like to do is make the templates specify a list of preprocessors to run.

Ideally we should be able to drop the SlidesExporter completely.

Reply:
I.e. have a new key in `conf.json` for `"prepropessors"`...

Reply:
Done.